whiteboards/tree-breadth-first/app.js

```diff
@@ -0,0 +1,17 @@
+function maxTrees(tree) {
```

Comment:
You might want to change the name to something like breadthFrist instead of maxTrees